modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -289,6 +289,11 @@ daemon set "multus" successfully rolled out
 
 . To complete changing the network plugin, reboot each node in your cluster. You can reboot the nodes in your cluster with either of the following approaches:
 
+[IMPORTANT]
+====
+The following scripts reboot all of the nodes in the cluster at the same time. This can cause your cluster to be unstable. Another option is to reboot your nodes manually one at a time. Rebooting nodes one-by-one causes considerable downtime in a cluster with many nodes.
+====
```

Comment:
Does the user know how to reboot nodes manually?

Reply:
I think so. The issue here is that we can't reboot gracefully. I checked with QE and another admin I _think_ this is a common enough task that we are ok.